src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -22,4 +22,30 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object rectangle) {
+        if (this == rectangle) {
+            return true;
+        }
+        if (rectangle == null || rectangle.getClass() != this.getClass()) {
+            return false;
+        }
+        Rectangle castedRectangle = (Rectangle) rectangle;
+        return (width == (castedRectangle.width)
```

Comment:
```suggestion
        return (width == castedRectangle.width
```
remove for other fields too

Reply:
removed braces